opentech/apply/activity/messaging.py

```diff
@@ -110,12 +111,19 @@ class ActivityAdapter(AdapterBase):
         MESSAGES.DETERMINATION_OUTCOME: 'Sent a determination. Outcome: {submission.determination.clean_outcome}',
         MESSAGES.INVITED_TO_PROPOSAL: 'Invited to submit a proposal',
         MESSAGES.REVIEWERS_UPDATED: 'reviewers_updated',
-        MESSAGES.NEW_REVIEW: '{user} submitted a review'
+        MESSAGES.NEW_REVIEW: '{user} submitted a review',
+        MESSAGES.OPENED_SEALED: '{user} opened the application while still sealed',
```

Comment:
should it not be "the submission" or "the application submission" ? Ditto for the base OPENED_SEALED message